GOVERNANCE.md

```diff
@@ -34,14 +31,31 @@ The Antrea community abides by this [code of conduct](CODE_OF_CONDUCT.md).
 Anyone can contribute to the project (e.g. open a PR) as long as they follow the
 guidelines in [CONTRIBUTING.md](CONTRIBUTING.md).
 
+Frequent contributors to the project can become members of the antrea-io Github
+organization and receive write access to the repository. Write access is
+required to trigger re-runs of workflows in [Github
+Actions](https://docs.github.com/en/actions/managing-workflow-runs/re-running-a-workflow). Becoming
+a member of the antrea-io Github organization does not come with additional
+responsibilities for the contributor, but simplifies the contributing
+process. To become a member, you may [open an
+issue](https://github.com/antrea-io/antrea/issues/new?template=membership.md&title=REQUEST%3A%20New%20membership%20for%20%3Cyour-GH-handle%3E)
+and your membership needs to be approved by two maintainers: approval is
+indicated by leaving a `+1` comment. If a contributor is not active for a
+duration of 12 months (no contribution of any kind), they may be removed from
+the antrea-io Github organization, In case of privilege abuse (members receive
+write access to the organization), any maintainer can decide to remove the
+member.
+
 ### Maintainers
 
 The list of current maintainers can be found in
 [MAINTAINERS.md](MAINTAINERS.md).
 
-Maintainers have write access to the repository. While anyone can review a PR -
-and is welcome to do so -, only maintainers can leave an approving review, which
-will allow the PR to be merged.
+While anyone can review a PR - and is encouraged to do so -, only maintainers
```

Comment:
I am not certain about "-,". Is it grammatically correct?

Reply:
It's supposed to be an em dash, which would be grammatically correct. But there is no native markdown way to print an em dash. I removed it altogether.